callbacks.py

```diff
@@ -56,6 +56,9 @@ def get_prediction_masks(self):
             X = data[0]
             targets_tensors = data[1:]
 
+            if (targets_tensors[0].size()[1] > 1):
+                targets_tensors = [target_tensor[:, :1, :, :] for target_tensor in targets_tensors]
```

Comment:
@apyskir I am pretty sure that python3 lets you do `target_tensor[: , :1]` here.

Reply:
@jakubczakon Yes, you're right.